app/build.gradle

```diff
@@ -16,12 +16,13 @@ android {
 
     namespace 'com.infomaniak.mail'
 
-    compileSdk 34
+    compileSdk 35
 
     defaultConfig {
         applicationId 'com.infomaniak.mail'
         minSdk 25
-        targetSdk 34
+        //noinspection EditedTargetSdkVersion
```

Comment:
Is this really necessary ?

Reply:
It all depends on your Android studio version

Reply:
I've updated but I don't have it anymore